client/webserver/site/src/js/markets.ts

```diff
@@ -2228,7 +2228,7 @@ class MarketList {
   /* exists will be true if the specified market exists. */
   exists (host: string, baseID: number, quoteID: number) {
     const xc = this.xcSection(host)
-    if (!xc) return false
+    if (!xc || !xc.marketRows) return false
```

Comment:
Could maybe use a comment as to why this is necessary so it doesn't get broken again in the future.

Reply:
Thanks for the review Joe, I added the comment.

Reply:
@JoeGruffins did you say you got an error on the wallets page too (as well as the markets page)?  It looks like @martonp fixed that in https://github.com/decred/dcrdex/commit/30e29d7107ac19ee8b9da5b4c701a18940ad898d#diff-1fb1a75f430311eb5c798880a2edbce5c4f529a823bd265bf255b3f4ba0cd8d9